scripts/dummy_script.py

```diff
@@ -0,0 +1,397 @@
+
+        elif order.order_side == TradeType.BUY:
+            self.buy(connector_name=connector_name, trading_pair=order.trading_pair, amount=order.amount,
+                     order_type=order.order_type, price=order.price)
+            # Add orders to the variable that keeps track of the number of orders on the exchange:
+            if connector_name == self.low_liquidity_exchange:
+                self.buy_orders_on_low_liquidity_exchange += 1
+                self.logger().info("Buy order placed on low liquidity exchange")
+            elif connector_name == self.high_liquidity_exchange:
+                self.sell_order_completed = False
+                self.logger().info("Buy order placed on high liquidity exchange")
+
+    # Methods concerning the high liquidity exchange
+    def calculate_spreads_and_volatility(self):
+        best_ask = self.connectors[self.high_liquidity_exchange].get_price(self.trading_pair, is_buy=True)
+        best_bid = self.connectors[self.high_liquidity_exchange].get_price(self.trading_pair, is_buy=False)
+
+        bid_ask_spread_bps = (best_ask - best_bid) * 10000
+
+        if bid_ask_spread_bps > 15:  # This can be adjusted
+            volatility = "high"
+        else:
+            volatility = "low"
+        return volatility
+
+    def is_active_maker_order(self, event: OrderFilledEvent):
+        """
+        Helper function that checks if order is an active order on the maker exchange
+        """
+        for order in self.get_active_orders(connector_name=self.maker_exchange):
+            if order.client_order_id == event.order_id:
+                return True
+        return False
+
+    def did_fill_order(self, event: OrderFilledEvent):
+        if event.trade_type == TradeType.BUY and self.is_active_maker_order(event):
+            self.buy_order_completed = True
+            self.logger().info(f"Buy order filled on low liquidity exchange with price: {event.price}")
+            hedge_order_amount = event.amount
+            return hedge_order_amount
+
+        if event.trade_type == TradeType.SELL and self.is_active_maker_order(event):
+            self.sell_order_completed = True
+            self.logger().info(f"Sell order filled on low liquidity exchange with price: {event.price}")
+            hedge_order_amount = event.amount
+            return hedge_order_amount
+
+    def determine_hedge_sell_order_price(self, volatility):
+        # We want the hedge to be placed on the best_ask. On_tick the bot will check if at this order.price the spread
+        # is too low (in case the order is a limit order).
+        best_ask = self.high_liquidity_exchange.get_price(self.trading_pair, is_buy= True)
+
+        if volatility == "high":
+            hedge_sell_order = OrderCandidate(trading_pair=self.trading_pair, is_maker=True, order_type=OrderType.LIMIT,
+                                              order_side=TradeType.SELL, amount=Decimal(self.did_fill_order()),
```

Comment:
this is an error, and I think that you have to manage the logic of send the market order in the event directly